sdk/tests/types/block.rs

```diff
@@ -105,25 +105,17 @@ fn getters() {
     let parents = rand_strong_parents();
     let payload = Payload::from(rand_tagged_data_payload());
 
-    let block = rand_basic_block_builder_with_strong_parents(protocol_parameters.clone(), parents.clone())
+    let block = rand_basic_block_builder_with_strong_parents(parents.clone())
         .with_payload(payload.clone())
         .finish()
         .unwrap();
+    let wrapper = rand_block_wrapper_with_block(protocol_parameters.clone(), block.clone());
```

Comment:
ew I don't like that these are separate now

Reply:
Changed a bit